packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -371,14 +369,24 @@ export const DatepickerCalendar: ComponentWithAs<'div', DatepickerCalendarProps>
                           }),
                       }),
                     )}
-                    {_.map(visibleGrid, week => renderWeekRow(week))}
-                  </>
-                ),
-              }),
-          },
-        )}
-      </ElementType>
-    </Ref>
+                  </tr>
+                </thead>
+                <tbody>
+                  {_.map(visibleGrid, (week, idx) =>
+                    createShorthand(DatepickerCalendarGridRow, calendarGridRow, {
+                      defaultProps: () =>
+                        getA11yProps('calendarGridRow', {
+                          children: renderWeekRow(week),
+                          key: idx + 1,
```

Comment:
Do we need the the hardcoded `key` here?

Reply:
Yeah, otherwise framework complains:
![image](https://user-images.githubusercontent.com/8701120/93470521-54ff1a00-f8f2-11ea-8de6-bc8161d025ca.png)